incubator/hnc/docs/user-guide/concepts.md

```diff
@@ -315,6 +316,40 @@ whichever labels they like. However, HNC will overwrite any changes made to
 these labels, so other applications can trust these labels for policy
 application.
 
+<a name="basic-exceptions"/>
+
+### Exceptions and propagation control
+
+By default, HNC propagates all objects of a [specified type](how-to.md/#admin-resources) 
+from ancestor namespaces to descendant namespaces. However, sometimes this is 
+too restrictive, and you need to create exceptions to certain policies. For example:
+
+A ResourceQuota was propagated to many children, but one child namespace now 
+has higher requirements than the rest. Rather than getting rid of the quota in 
+the parent namespace, or raising the limit for everyone, you can stop the 
+quota in the parent from being propagated to that one child namespace, 
+allowing you to replace it with another, more suitable quota.
+
+A RoleBinding allows any user to create subnamespaces under one namespace, but 
+we don’t want to allow those users to create additional levels of hierarchy 
+underneath those subnamespaces. So you can stop the role binding from being 
+propagated to any child namespace.
+
+Exceptions are defined using [annotations on the objects themselves](how-to.md/#use-limit-propagation). 
+As a result, anyone who can edit an object can also control how it is 
+propagated to descendant namespaces.
+
+If you modify an exception - for example - by removing it - this could cause 
```

Comment:
Whoops that second dash should be a comma: "If you modify an exception - for example, by removing it - this could..."